cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/common/MetadataClient.java

```diff
@@ -89,10 +89,23 @@ public synchronized ClusterAndGeneration refreshMetadata(long timeout) {
     return new ClusterAndGeneration(_metadata.fetch(), _metadataGeneration.get());
   }
 
+  /**
+   * Get the metadata maintained by this metadata client.
```

Comment:
This is kind of useless documentation.  For methods like getX saying that it returns X is not really useful.  Saying something like "X is non-null" or "X may be null" or something like that is useful.

Reply:
Yeah, I guess it is more of follow the convention to have a Java doc for each public method...